test/integration/async-modules/test/index.test.js

```diff
@@ -101,6 +101,19 @@ function runTests(dev = false) {
       if (browser) await browser.close()
     }
   })
+  ;(dev ? it.skip : it)('compiles the source to es5', async () => {
+    const html = await renderViaHTTP(appPort, '/')
+
+    const appScriptMatch = html.match(
+      /\/_next\/static\/chunks\/pages\/_app-.+?\.js/
+    )
```

Comment:
Not sure if there's a way to avoid hardcoding this path or to generally rewrite this test. For now I only tried to get a failing test that demonstrates the issue.